plugins/modules/azure_rm_virtualmachine_info.py

```diff
@@ -386,6 +390,7 @@ def serialize_vm(self, vm):
         new_result['state'] = 'present'
         new_result['location'] = vm.location
         new_result['vm_size'] = result['properties']['hardwareProfile']['vmSize']
+        new_result['zones'] = result['zones']
```

Comment:
It will return null when the VM not config zones!
```suggestion
        new_result['zones'] = result.get('zones', None)
```

Reply:
Good catch!